NOTICE

```diff
@@ -65,29 +69,9 @@ IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 --------------------------------------------------------------------
-github.com/cavaliercoder/badio
```

Comment:
I excluded the `dev-tools` folder, since those are not run-time dependecies. That one is needed for the test we have on packages.